### PR TITLE
#484: multiple call update removed

### PR DESCRIPTION
**Pull request by phong-steph**

### CODE REVIEW

File: managers/software.js [hunk 1]
```diff
@@ -34,19 +34,12 @@ module.exports.allSoftwaresByUser = function (user, socket) {
...
-          });
-        }).catch(function (err){
-          socket.emit('server_all_software_by_user', err);
-          logger.setUser({ id: '', name: user }).error(`${user} failed to update ${package}`);
-        });
+        socket.emit('server_all_software_by_user', package);
       });
-  });
+    }).catch(function (err){
+      socket.emit('server_all_software_by_user', err);
```
Comment (BernardJeremy) [inline managers/software.js:40]: Does the webapp handle error in this case ?

Comment (clairealmozinos) [reply managers/software.js:40]: @BernardJeremy it will be solved with #489.

Comment (BernardJeremy) [reply managers/software.js:40]: okay

**Review by BernardJeremy — APPROVED** [review]

**Comment by phong-steph** [comment]
https://github.com/EIP-SAM/SAM-Solution-Server/issues/484

**Pull request merged**